src/raw_statement.rs

```diff
@@ -69,6 +69,24 @@ impl RawStatement {
         unsafe { ffi::sqlite3_column_type(self.ptr, idx as c_int) }
     }
 
+    #[inline]
+    pub fn column_table_name(&self, idx: usize) -> Option<&CStr> {
+        let idx = idx as c_int;
+        if idx < 0 || idx >= self.column_count() as c_int {
+            return None;
+        }
+
+        unsafe {
+            let ptr = ffi::sqlite3_column_table_name(self.ptr, idx);
```

Comment:
`sqlite3_column_table_name` depends on `SQLITE_ENABLE_COLUMN_METADATA` compile option.
So you must introduce a feature.